app/main.js

```diff
@@ -89,6 +89,10 @@
 
     async function initTheme() {
         F.util.chooseTheme(await F.state.get('theme', 'default'));
+        const orgStylesheet = (await F.currentUser.getOrg()).get('custom_css');
+        if (orgStylesheet) {
+            $(`<style type="text/css">${orgStylesheet}</style>`).appendTo('head');
```

Comment:
I think with this technique the user could perform html injection but including a closing </style> tag, no?  Might be necessary to create the style element and inject the contents into it as text. vis $(myStyleElement).text(orgStylesheet);